config/controllers_deployment.yml

```diff
@@ -43,11 +43,11 @@ spec:
           value: #@ data.values.workloads_namespace
         resources:
           limits:
-            cpu: 100m
-            memory: 200Mi
+            cpu: 1000m
+            memory: 1Gi
```

Comment:
The `SharedInformer` cache for this controller needs memory to be able to load information about all the CRDs its watching. For larger environments with a lot of routes and/or kpack builds/images it's very likely for it to get OOMKilled with a limit at 200mb.